Makefile

```diff
@@ -898,6 +899,17 @@ ifeq ($(CONFIG_WIFI_MONITOR), y)
 EXTRA_CFLAGS += -DCONFIG_WIFI_MONITOR
 endif
 
+ifeq ($(CONFIG_PLATFORM_ARM_RPI), y)
+    EXTRA_CFLAGS += -DCONFIG_LITTLE_ENDIAN
+    EXTRA_CFLAGS += -DCONFIG_IOCTL_CFG80211 -DRTW_USE_CFG80211_STA_EVENT
+    ARCH ?= arm
+    CROSS_COMPILE ?=
+    KVER ?= $(shell uname -r)
+KSRC := /lib/modules/$(KVER)/build
+MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/
```

Comment:
I believe these settings should be provided by the user, and should not be preset by the Makefile